src/store/actionCreators/Test/fenceConnections.test.ts

```diff
@@ -61,8 +53,11 @@ describe('Fence Connections actions', () => {
   });
 
   it('should manage acls adequately', () => {
+    // @ts-ignore
```

Comment:
why do you need to ignore ts here ?

Reply:
`ts-ignore` requires an detailed explanation and justification

Reply:
@celinepelletier remember the Nullable? That's why I need a ts-ignore, because, as received from the network, some fields can be null. However in the Connection Type def I didn't say that a field can be null.

![image](https://user-images.githubusercontent.com/54366437/125672006-bbd11c49-128a-4900-93f1-9f2135b6064a.png)

Reply:
is it a real use case that API returns null values ?

Reply:
When I log the data received from a fence - I get something similar to the mock object (a lot of null values in my case).

However, I can just remove those fields in the mock object.

Reply:
ok so you could declare your property as : 
`display_name?: string | null`
It should do the job

Reply:
it would. That's why I used Nullable last time as a kinda abbreviation. For the tests I just removed the null fields from the mock object.